app/js/services/individuService.js

```diff
@@ -1,7 +1,7 @@
 'use strict';
 
 angular.module('ddsCommon').service('IndividuService', function($filter, situationsPro, nationalites) {
-    var statutsSpecifiques = _.filter(situationsPro, 'isStatutSpecifique');
+    var statutsSpecifiques = situationsPro;
```

Comment:
Well, looking at the whole file there are weird `find` in this array. Removing the intermediate variable would improve readability and possibly simplify filters.